### ISSUE TITLE
coursera/week5_policy_based/practice_reinforce.ipynb does not call xvfb

### ISSUE THREAD

**Issue by Pastafarianist** [yandexdataschool/Practical_RL#121]
Need to add a standard preamble:

```python
import numpy as np
import matplotlib.pyplot as plt
%matplotlib inline

# This code creates a virtual display to draw game images on. 
# If you are running locally, just ignore it
import os
if type(os.environ.get("DISPLAY")) is not str or len(os.environ.get("DISPLAY"))==0:
    !bash ../xvfb start
    %env DISPLAY=:1
```

Another way is to get rid of IPython magics using something like this:

```python
import os
import subprocess

if not os.environ.get('DISPLAY', ''):
    subprocess.check_call(['bash', '../xvfb', 'start'])
    os.environ['DISPLAY'] = ':1'
```

**Comment by Pastafarianist** [comment]
Postponing this till the grand merge `spring19` → `coursera`.

**Comment by yhn112** [comment]
Fixed in #243
